requirements.txt

```diff
@@ -10,3 +10,4 @@ python-memcached>=1.56 # PSF
 PyYAML>=3.12 # MIT
 virtualenv>=14.0.6 # MIT
 shade>=1.26.0
+configparser>=3.5.0
```

Comment:
I couldn't get this to work - and using the six try/except import will mean this should work with py3 and py2